pandas/tests/io/json/test_normalize.py

```diff
@@ -123,6 +123,12 @@ def test_simple_normalize_with_separator(self, deep_nested):
                           'country', 'states_name']).sort_values()
         assert result.columns.sort_values().equals(expected)
 
+    def test_value_array_record_prefix(self):
+        # GH 21536
+        result = json_normalize({'A': [1, 2]}, 'A', record_prefix='Prefix.')
+        expected = DataFrame([[1], [2]], columns=['Prefix.0'])
+        tm.assert_frame_equal(result.reindex_like(expected), expected)
```

Comment:
Don't think `reindex_like` is required here?

Reply:
Yes. Don't need it. I followed the code in `test_simple_normalize_with_separator`.
Btw, do we need docstring for test functions? None of the functions in this file has docstring.

Reply:
Ah okay. Done. Thanks.